CONTRIB.md

```diff
@@ -1,33 +1,128 @@
+2. Create a branch for your story by running `git checkout -b '[kebab-case-story-name]'`. Replace the values in brackets with the relevant names. Read more about kebab-case [here](https://www.freecodecamp.org/news/programming-naming-conventions-explained#what-is-kebab-case).
+3. In the terminal run `npm run create-visual --year=[year] --story=[kebab-case-story-name]` This will create a folder with all the necessary files to create your visual.
+    - If it is the year `2024` and your story is called `Example Visual` then run the command `npm run create-visual --year=2023 --story=example-visual` using the year `2024` and your story's name `example-visual` in kebab-case.
+    - For the error `The syntax of the command is incorrect`, use ChatGPT to edit the command for your terminal. For example, Git Bash requires line breaks. You may also have to manually set environment variables with `npm_config_year=` and `npm_config_story=`.
+    - In your `./[year]/[story-name]/` folder, there will be a file called `main.js`. This is where you’ll write the JavaScript logic for your visual.
+4. Start creating your visual that will be housed in the `./[year]/[story-name]/` folder.
+    - If you'd like to see previously created example visuals, open [example-graphs.html](./example-graphs.html) from your local repository.
+    - If you are using Highcharts, here is a lot of helpful documentation on the [options for creating charts](https://api.highcharts.com/highmaps/).
+5. Update the `meta_data.json` in your story's folder, which will look something like this:
+    ```json
+    {
+        "title": "Example Visual",
+        "description": "Describe your visual"
+    }
+    ```
+    - Make sure the title you write in the `meta_data.json` file is the same as the title you give your visual in `main.js` and in `index.html`'s `<title>` tag.
+6. Run `npm run lint` and make the changes that the output recommends.
+7. Go to the base folder of the `data-visualizations` repository and run the command `npm run process-visual --year=[year] --story=[kebab-case-story-name]`
+    - Example: `npm run process-visual --year=2023 --story=example-visual`
+8. Create a Pull Request to have your visual reviewed by committing and pushing your changes.
+    - You can run `git status` to see which files in red have not been added. Then run `git add [file-name]` for each file to add them. Next run `git commit -m '[commit message]'` to commit your changes. Then run `git push` to push your changes. If this is your first commit, github may ask you to run a different command, which you should. Ex `git push --set-upstream origin [branch-name]`
+
+## The Chicago Maroon’s Data Contribution and Style Guide
+
+### Contribution Standards
+
+#### Pull Request Titles
 
 They should be written in the present imperative tense. Examples of that tense are:
 "Fix issue in the dispatcher where…", "Improve our handling of…", etc."
```

Comment:
PR titles are truncated at 70 characters. I'd add something about that in here as well as the PR template.

Reply:
i dont think anyone will be counting 70 characters, if its too long, a v quick fix in github